llama.h

```diff
@@ -759,6 +759,12 @@ extern "C" {
     // n_threads_batch is the number of threads used for prompt and batch processing (multiple tokens)
     LLAMA_API void llama_set_n_threads(struct llama_context * ctx, uint32_t n_threads, uint32_t n_threads_batch);
 
+    // Get the number of threads used for generation of a single token.
+    LLAMA_API uint32_t llama_get_n_threads(struct llama_context * ctx);
+
+    // Get the number of threads used for prompt and batch processing (multiple token).
+    LLAMA_API uint32_t llama_get_n_threads_batch(struct llama_context * ctx);
```

Comment:
Most of the llama_context getters (such as llama_n_ctx) do not have "get" in the name. I think naming these llama_n_threads and llama_n_threads_batch would be more consistent.

Reply:
Ah that makes sense, I'll update shortly. Thanks!

Reply:
That makes sense, I've updated it now. Thanks!